docs/syntax/pattern.md

```diff
@@ -6,6 +6,8 @@ Patterns provide convenient ways to manipulate data.
 Literal Patterns
 ------------------------
 
+Patterns with a literal (e.g. `1`, `false`, `nothing`, `[4.0, 1f-6]`, etc) on the left hand side will check if the the argument is equal to that literal:
```

Comment:
`[4.0, 1f-6]` is actually an array pattern

Reply:
Oops, I was thinking about how people call that an "array literal" in julia, but this is not what you mean in this package by literal.

Reply:
Fixed.